internal/controller/controllers/preprovisioningimage_controller.go

```diff
@@ -590,7 +590,7 @@ func (r *PreprovisioningImageReconciler) getIronicConfig(ctx context.Context, lo
 		return nil, fmt.Errorf("Failed to determine ironic config")
 	}
 
-	log.Infof("Ironic Agent Image is (%s) Ironic URL is (%s) Inspector URL is (%s)",
+	log.Debugf("Ironic Agent Image is (%s) Ironic URL is (%s) Inspector URL is (%s)",
```

Comment:
This is logic we've had issues with in the past instead of not logging this could we maybe keep it as info, but move it into the block in the caller where we know the config changed so we need to update it?